asmcli/components/ca/managed-cas.sh

```diff
@@ -92,7 +92,7 @@ x_wait_for_enabling_workload_certificates() {
       -H "Authorization: Bearer $(gcloud auth print-access-token)" \
       "https://${GKEHUB_API}/v1alpha/projects/${FLEET_ID}/locations/global/features/workloadcertificate" 2>/dev/null | \
       grep -i "certificateManagement.*ENABLED")"
-    if [[ -z "$OUT" ]]; then
+    if [[ -n "$OUT" ]]; then
```

Comment:
Why changes -z to -n here?

Reply:
IIUC, this condition should check if OUT is not nil. That is accomplished with the -n flag? the -z flag is the opposite?

Reply:
The -z flag checks the string is empty or not, whereas -n checks a variable is nil or not. A non-nil variable may have an empty string. So "-z" should be used here.